app/javascript/controllers/sticky_bar_controller.js

```diff
@@ -0,0 +1,23 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  connect () {
+    this.toggle_sticky_shadow_on_scroll()
+
+    window.addEventListener('scroll', () => this.toggle_sticky_shadow_on_scroll())
+  }
+
+  disconnect () {
+    window.removeEventListener('scroll', () => this.toggle_sticky_shadow_on_scroll())
+  }
+
+  toggle_sticky_shadow_on_scroll () {
```

Comment:
je suis aussi étonné que cela passe le linter

```suggestion
  toggleStickyShadowOnScoll () {
```